src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -30,6 +30,21 @@ class JwtAuthenticator implements AuthenticatorInterface
      */
     private $jwt;
 
+    /**
+     * @var string
+     */
+    private $jws = '';
+
+    /**
+     * @var integer
+     */
+    private $jwsLifetime = 3600;
+
+    /**
+     * @var integer
+     */
+    private $jwsCreatedAt = 0;
```

Comment:
Why the timestamp? The timestamp is a very hard for debugging because the developer should convert the timestamp to human-readable values.

Maybe better use `\DateTime` instances?